conf/messages.en

```diff
@@ -1325,4 +1325,10 @@ goodsTooLargeForContainerYesNo.error.required = Select yes if the goods are too
 sealsReplacedByCustomsAuthorityYesNo.title = Have any seals been replaced by a customs authority?
 sealsReplacedByCustomsAuthorityYesNo.heading = Have any seals been replaced by a customs authority?
 sealsReplacedByCustomsAuthorityYesNo.paragraph = This could have happened during routine checks or following an incident.
-sealsReplacedByCustomsAuthorityYesNo.error.required = Select yes if any seals been replaced by a customs authority
\ No newline at end of file
+sealsReplacedByCustomsAuthorityYesNo.error.required = Select yes if any seals been replaced by a customs authority
```

Comment:
Doesn't look like your change but this sentence doesn't read well - is it correct?